packages/dds/tree/src/core/rebase/utils.ts

```diff
@@ -74,6 +89,18 @@ export interface BranchRebaseResult<TChange> {
 	 * Details about how the commits on the source branch changed
 	 */
 	readonly commits: RebasedCommits<TChange>;
+	/**
+	 * Telemetry properties for the rebase operation.
+	 */
+	readonly telemetryProperties?: TelemetryProperties;
+}
+
+interface RebaseChangeResult<TChange> {
```

Comment:
Curious if:
- If it's alright to create a new return type for `rebaseChange()`.
- If it is, whether or not the name of the type is intuitive.

Reply:
Yes and yes